template.yaml

```diff
@@ -41,6 +41,16 @@ Parameters:
     Type: String
     Description: Amazon DynamoDB Stream ARN.
 
+  EventBusName:
+    Type: String
+    Default: '{{resolve:ssm:nva-eventbus-name:1}}'
```

Comment:
Can we use https://aws.amazon.com/blogs/mt/integrating-aws-cloudformation-with-aws-systems-manager-parameter-store/

Reply:
Sounds like a good idea. +1

Reply:
fixed and tested